index.html

```diff
@@ -318,6 +318,12 @@ <h3>
     <li>
       a <dfn>[[\SampleRate]]</dfn> double set on creation.
     </li>
+    <li>
+      a <dfn>[[\PendingObservePromises]]</dfn> [=list=] of zero or more source-promise [=tuples=], initially empty,
+      where source holds a {{PressureSource}} string and promise holds a {{Promise}} object.
+      When a {{Promise}} object contained in a [=tuple=] settle (resolve or rejects), the
+      associated [=tuple=] is removed.
```

Comment:
I don't see the actual removal in the algorithms below; don't you need to do that normatively?

Reply:
I think this is good enough for now. WebHID doesn't do it either.

If I can find a way to spec this nicely, I can make another PR. I will probably have to consult with Domenic

Reply:
WebHID does it partially: https://wicg.github.io/webhid/#dfn-pendingsendreportpromises is emptied in https://wicg.github.io/webhid/#ref-for-dfn-pendingsendreportpromises-2 and https://wicg.github.io/webhid/#ref-for-dfn-pendingsendreportpromises-4 but not in the success case.

Web Bluetooth seems to handle it a bit better: https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-attributeinstancemap-slot

Even it's really necessary to do it separately, please file an issue so we don't lose track of it. The current text is too hand-wavy from an implementation perspective so more clarity will be needed.

Reply:
Sure, filed https://github.com/WICG/compute-pressure/issues/168